src/components/questionStructure/Task/variants/HandwritingTask/HandwritingEditor.tsx

```diff
@@ -5,27 +5,33 @@ import React from 'react'
 import Markdown from '../../../../Markdown'
 import Canvas from './Canvas'
 import './handwritingEditor.css'
+import { HandwritingAnswer } from './types'
 
 interface HandwritingEditorProps {
   username: string
-  latex: string
+  answer?: HandwritingAnswer
   onAnswerChange: (value: string) => void
 }
 
 const HandwritingEditor: React.FC<HandwritingEditorProps> = ({
   username,
-  latex,
+  answer,
   onAnswerChange,
 }) => {
   return (
     <Flex direction="column" flexGrow="1" align="center" gap="3">
       <Card className="mathjax-card">
         <Box p="3">
-          <MathJax>{`\\( ${latex} \\)`}</MathJax>
+          <MathJax>{`\\( ${answer?.latex} \\)`}</MathJax>
         </Box>
       </Card>
       <Card className="excalidraw-canvas-card">
-        <Canvas username={username} onAnswerChange={onAnswerChange} />
+        <Canvas
+          username={username}
+          onAnswerChange={onAnswerChange}
+          // @ts-expect-error: this is not just initialData
+          initialData={answer?.excalidraw ?? {}}
```

Comment:
Could we call it something other than `initialData` then? Like `strokes` (which sounds more on point and precise for a `Canvas`).

Reply:
The thing is that this is just passed straight to Excalidraw itself and _only_ used on first render (rather than as a reactive state) - perhaps `initialStrokes` makes more sense then?

Reply:
Mhmh, ok yes. Why the `ts-expect-error` then?

Reply:
The type of what counts as "current data" (i.e. what gets exported from excalidraw on every update) is different to the type of the initial data, but everything still works fine, so the expect-error just tell TypeScript we know what we're doing even if the types don't exactly match.

Reply:
(we'll add this to the ts-expect-error comment)

Reply:
OK